avocado-setup.py

```diff
@@ -377,7 +378,12 @@ def run_test(testsuite, avocado_bin):
     logger.info('')
     result_link = testsuite.jobdir()
     if result_link:
-        result_link += "/job.log"
+        result_json = result_link + "/results.json"
+        result_link += "/job.log\n"
+        with open(result_json, encoding = "utf-8") as fp:
+            result_state = json.load(fp)
+        for state in ['total', 'pass', 'cancel', 'errors', 'failures', 'skip']:
```

Comment:
please add `warn` as well

Reply:
@harish-24 `WARN` is not found in the json file. So I did not add it.

Reply:
@narasimhan-v I see a warn in there,
```
{
    "cancel": 0,
    "debuglog": "/home/satheesh/data/gits/github/avocado/job-2020-03-30T12.39-8a51c2c/job.log",
    "errors": 0,
    "failures": 0,
    "interrupt": 0,
    "job_id": "8a51c2c2ab2e400fc5089b3e6fd26244e383427e",
    "pass": 1,
    "skip": 0,
    "tests": [
        {
            "end": 1585552154.4747539,
            "fail_reason": null,
            "id": "1-/bin/true",
            "logdir": "/home/satheesh/data/gits/github/avocado/job-2020-03-30T12.39-8a51c2c/test-results/1-_bin_true",
            "logfile": "/home/satheesh/data/gits/github/avocado/job-2020-03-30T12.39-8a51c2c/test-results/1-_bin_true/debug.log",
            "start": 1585552154.219356,
            "status": "PASS",
            "time": 0.2553977966308594,
            "whiteboard": ""
        }
    ],
    "time": 0.2553977966308594,
    "total": 1,
    "warn": 0
```

Reply:
okay, let me recheck and add this. Thanks @harish-24 and @sathnaga

Reply:
Seems I had older avocado in the system I tested.
Will add them.
```
commit c6b0c8180cd38b66d29729eb825bc0d4155a33c8
Author: Wainer dos Santos Moschetta <wainersm@redhat.com>
Date:   Fri Jan 10 15:25:48 2020 -0300

    plugins: jsonresult: add missing warn and interrupt fields
```

Reply:
@harish-24 please check now. Added warn and interrupt